### PR TITLE
3748 sortable index tables

### PR DESCRIPTION
**Pull request by emitche**
Adds tables to application. Adds simple organizations table with sortable columns.

### CODE REVIEW

File: app/views/admin/organizations/index.html.slim [hunk 1]
```diff
@@ -0,0 +1,24 @@
+- content_for :title do
+  - 'Co-ops and Organizations'
```
Comment (linesarefuzzy) [inline app/views/admin/organizations/index.html.slim]: Just "Co-ops" for now

Comment (emitche) [inline app/views/admin/organizations/index.html.slim]: Should the link be coops to avoid confusion between coops and lending organizations?

**Comment by linesarefuzzy** [comment]
I'm getting a NoMethodError when there are no orgs in my db. It would be great if it checked somewhere if the set is empty and showed a message rather than an error.

File: app/views/admin/organizations/index.html.slim [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - g.column name: 'City', attribute: 'city'
+
+  - g.column name: 'Country' do |organization|
+    - organization.country.name
+
+  - g.column name: 'Created', attribute: 'created_at' do |organization|
+    - organization.created_at
+
+  - g.column name: 'Updated', attribute: 'updated_at' do |organization|
+    - organization.updated_at
```
Comment (linesarefuzzy) [inline app/views/admin/organizations/index.html.slim]: Why have blocks for created and updated?

File: config/initializers/wice_grid_config.rb [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+  Wice::Defaults::DEFAULT_TABLE_CLASSES = ['table', 'table-bordered', 'table-striped']
+
+  # Allow switching between a single and multiple selection modes in custom filters (dropdown boxes)
+  Wice::Defaults::ALLOW_MULTIPLE_SELECTION = true
+
+  # Show the upper pagination panel by default or not
+  Wice::Defaults::SHOW_UPPER_PAGINATION_PANEL = false
+
+  # Disabling CSV export by default
+  Wice::Defaults::ENABLE_EXPORT_TO_CSV = false
```
Comment (linesarefuzzy) [inline config/initializers/wice_grid_config.rb]: Built in CSV export = awesome!

File: config/locales/wice_grid.yml [hunk 4]
```diff
@@ -0,0 +1,636 @@
...
+    saved_query_deletion_link_title: Vymazať uložený filter
+    saved_query_link_title: Nahrať uložený filter
+    validates_uniqueness_error: Filter s uvedeným názvom už existuje.
+    validates_presence_error: Prosím zadaj názov uloženého filtra.
+    query_deleted_message: Uložený filter bol vymazaný.
+    query_saved_message: Filter bol uložený.
+    select_all: Vyber všetko
+    deselect_all: Zrušiť výber
+
+cz:
```
Comment (linesarefuzzy) [inline config/locales/wice_grid.yml]: Translated to Czech. That's a relief. :P

File: app/views/admin/organizations/index.html.slim (hunk 2 above)
Comment (emitche) [inline app/views/admin/organizations/index.html.slim]: For date formatting.

Per Fuzzy, wice grid can handle default date formats. So blocks may not be needed.

File: config/locales/wice_grid.yml (hunk 4 above)
Comment (emitche) [inline config/locales/wice_grid.yml]: Automated.

File: config/initializers/wice_grid_config.rb (hunk 3 above)
Comment (emitche) [inline config/initializers/wice_grid_config.rb]: We want this though, so later we'll enable, right? Very cool and saves time.

**Comment by emitche** [comment]
Per Fuzzy, if no organizations, instead of displaying empty table or possibly getting an error, should display "No organizations found."

**Comment by linesarefuzzy** [comment]
Looks great! Merge away when that empty message is done, unless you want me to review it again.

**Pull request merged**